modules/preview-react/radio/stories/examples/Custom.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import {FormField} from '@workday/canvas-kit-react/form-field';
+import {RadioGroup} from '@workday/canvas-kit-preview-react/radio';
+import {Flex} from '@workday/canvas-kit-react/layout';
+
+export const Custom = () => {
+  const [value, setValue] = React.useState<string | number>('deep-dish');
+
+  const handleChange = (e: React.ChangeEvent) => {
+    const target = e.currentTarget;
+    if (target instanceof HTMLInputElement) {
+      setValue(target.value);
+    }
+  };
+
+  return (
+    <Flex flexDirection="column">
+      <FormField label="Choose Your Pizza Crust" useFieldset={true}>
+        <RadioGroup
+          name="pizza-crust-custom"
+          onChange={handleChange}
+          width="200px"
+          value={value}
+          gap="m"
+        >
```

Comment:
Suggest removing the `gap="m"` in the Custom example since it's not relevant to using `RadioGroup.Label`. It may distract from what we mean by "custom" here (though it is handy that `RadioGroup` accepts flex props).

```suggestion
          value={value}
        >
```